server/deploy/kubernetes/db-mariadb/db-deploy.yaml

```diff
@@ -36,11 +36,17 @@ spec:
       containers:
       - name: mariadb
         image: docker.io/centos/mariadb-102-centos7:latest
+        lifecycle:
+          postStart:
+            exec:
+              command: ["/bin/sh", "-c", "sleep 60; scl enable rh-mariadb102 'mysql -u root teamtalk' < /var/lib/mysql/ttopen.sql"]
```

Comment:
If the database already has data and the pod recreates, this will delete all data from the database.
60 second sleep seems random to me.